configure.ac

```diff
@@ -385,6 +385,10 @@ SECP_CFLAGS="$SECP_CFLAGS $WERROR_CFLAGS"
 ### Handle module options
 ###
 
+# Besides testing whether modules are enabled, the following code also enables
+# module dependencies. The order of the tests matters: the dependency must be
+# tested first.
+
```

Comment:
nit: is the comment worth adding to upstream?

Reply:
no imo, as there are comments already (what I removed here)